index.html

```diff
@@ -228,6 +228,12 @@ <h1 class="hero-header">
           <p>
             If you need more information on voting or the election process, visit <a href="https://www.usa.gov/voting" target="blank">USA.gov/voting</a>. To find this information in Spanish, visit <a href="https://gobierno.usa.gov/elecciones-electorales" target="blank">GobiernoUSA.gov</a>.
           </p>
+          <p>
+            To learn more about our current progress &mdash; and to make suggestions &mdash;
+            visit our <a
+              href="https://github.com/presidential-innovation-fellows/vote-2016"
+              target="_blank">GitHub repository</a>.
```

Comment:
Britta from the #wg-opensource team asked me to remove reference to GitHub in another project (Apps.gov) with a replacement of 'source code repository' or some other plain-language vendor-neutral link name